client/src/components/RankingClass.vue

```diff
@@ -29,8 +29,8 @@ const props = defineProps<{
 
 const displayedDescription = computed(() =>
   props.short
-    ? props.rankingClass.shortDescription
-    : props.rankingClass.description
+    ? props.rankingClass?.shortDescription
```

Comment:
Wieso fällt das erst jetzt auf?

Reply:
Weil jetzt die Typen richtig importiert wurden in der `tsconfig.json`=> `  "include": ["env.d.ts", "components.d.ts", "src/**/*", "src/**/*.vue"],
`